website/docs/r/redshift_snapshot_copy_grant.html.markdown

```diff
@@ -0,0 +1,118 @@
+---
+layout: "aws"
+page_title: "AWS: aws_redshift_snapshot_copy_grant"
+sidebar_current: "docs-aws-resource-redshift-snapshot-copy-grant"
+description: |-
+  Creates a snapshot copy grant that allows AWS Redshift to encrypt copied snapshots with a customer master key from AWS KMS in a destination region.
+---
+
+# aws_redshift_snapshot_copy_grant
+
+Creates a snapshot copy grant that allows AWS Redshift to encrypt copied snapshots with a customer master key from AWS KMS in a destination region.
+
+Note that the grant must exist in the destination region, and not in the region of the cluster.
+
+## Example Usage
+
+```hcl
+resource "aws_redshift_snapshot_copy_grant" "test" {
+    snapshot_copy_grant_name = "my-grant"
+}
+
+resource "aws_redshift_parameter_group" "test" {
```

Comment:
We prefer to only show very relevant example configuration in our documentation in case other resources/attributes/implementations change, e.g.

```hcl
resource "aws_redshift_snapshot_copy_grant" "example" {
  snapshot_copy_grant_name = "example"
}

resource "aws_redshift_cluster" "example" {
  # ... other configuration ...
  snapshot_copy {
    destination_region = "us-east-2"
    grant_name         = "${aws_redshift_snapshot_copy_grant.example.snapshot_copy_grant_name}"
  }
}
```